web-ng/src/app/components/main-page/main-page.component.ts

```diff
@@ -69,6 +70,11 @@ export class MainPageComponent implements OnInit {
       return;
     }
     const params = new HttpParams({ fromString: fragment });
+    // The 'l' param with value ':new" is used to create new layer.
```

Comment:
If this logic is handled inside showEditLayerDialog(), this block can be deleted.